build.gradle

```diff
@@ -13,10 +13,11 @@ repositories {
     maven { url 'https://s01.oss.sonatype.org/content/repositories/releases' }
     maven { url 'https://s01.oss.sonatype.org/content/repositories/snapshots/' }
     maven { url 'https://repo.spring.io/milestone' }
+    mavenLocal()
 }
 
 dependencies {
-    implementation platform('run.halo.tools.platform:plugin:2.17.0-SNAPSHOT')
+    implementation platform('run.halo.tools.platform:plugin:2.20.0-LOCAL')
```

Comment:
Change the version to 2.20.0-SNAPSHOT after merging <https://github.com/halo-dev/halo/pull/6670>.